ivy/stateful/module.py

```diff
@@ -448,6 +491,17 @@ def _create_variables(self, *, device=None, dtype=None):
         """
         return {}
 
+    def _create_buffers(self):
```

Comment:
Probably this is just left over from the previous changes, can we remove this method too? Users would be able to create buffers at any point by just calling `self.register_buffer`.
Also, could you please message on the PR once you think it's ready for review? Just to avoid me reviewing the PR if it isn't supposed to be reviewed. Thanks for digging into this @RickSanchezStoic 😄💪

Reply:
yep